care/facility/tasks/plausible_stats.py

```diff
@@ -0,0 +1,131 @@
+import logging
+from datetime import timedelta
+from enum import Enum
+
+from celery import shared_task
+from django.conf import settings
+from django.utils.timezone import now
+from requests import request
+
+from care.facility.models.stats import Goal, GoalEntry, GoalProperty, GoalPropertyEntry
+
+logger = logging.getLogger(__name__)
+
+
+class Goals(Enum):
+    PATIENT_CONSULTATION_VIEWED = ("facilityId", "consultationId", "userId")
+    DOCTOR_CONNECT_CLICKED = ("consultationId", "facilityId", "userId", "page")
+    CAMERA_PRESET_CLICKED = ("presetName", "consultationId", "userId", "result")
+    CAMERA_FEED_MOVED = ("direction", "consultationId", "userId")
+    PATIENT_PROFILE_VIEWED = ("facilityId", "userId")
+    DEVICE_VIEWED = ("bedId", "assetId", "userId")
+    PAGEVIEW = ("page",)
+
+    @property
+    def formatted_name(self):
+        if self == Goals.PAGEVIEW:
+            return "pageview"  # pageview is a reserved goal in plausible
+        return self.name.replace("_", " ").title()
+
+
+def get_goal_stats(plausible_host, site_id, date, goal_name):
+    goal_filter = f"event:name=={goal_name}"
+    url = f"https://{plausible_host}/api/v1/stats/aggregate?site_id={site_id}&filters={goal_filter}&period=day&date={date}&metrics=visitors,events"
+
+    response = request(
+        "GET",
+        url,
+        headers={
+            "Authorization": "Bearer " + settings.PLAUSIBLE_AUTH_TOKEN,
+        },
+    )
+
+    return response.json()
```

Comment:
- [ ] use request.get()
- [ ] set a timeout for the api call
- [ ] call request.raise_for_status()